projects/Mallard/src/screens/home-screen.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
-import { View } from 'react-native'
+import { View, Text } from 'react-native'
 import {
     NavigationInjectedProps,
     NavigationScreenProp,
     withNavigation,
+    FlatList,
```

Comment:
Ooops, need to use `FlatList` from react native, not react-navigation!!!

Reply:
Blocking this PR so this doesnt get merged. We had issues of using FlatLists from React Navigation in the past